src/node/coinstats.cpp

```diff
@@ -15,6 +15,7 @@
 
 #include <map>
 
+// Database-independent, meaningless metric indicating the UTXO set size
```

Comment:
```suggestion
// Database-independent metric indicating the UTXO set size
```

nit: It means to represent the size, so not meaningless

Reply:
done